src/components/Search/SearchOutputGrid.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="container-fluid">
+        <div class="row">
+            <div v-for="(record, index) in input_data" :key="index" class="col col-xl-4 col-lg-4 col-sm-6">
+                <div class="card">
+                    <div class="card-header">
+                        <a v-bind:href="'#/'+record.id">{{record.name}}</a>
+                    </div>
+                    <div class="card-body">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 spaces but found 20 spaces.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)